### PR TITLE
Bugfix change golem configuration directory to use absolute path

### PR DESCRIPTION
**Pull request by rwrzesien**
Resolves https://github.com/golemfactory/concent-deployment/issues/322.

### CODE REVIEW

**Comment by cameel** [comment]
@rwrzesien You have your fix mixed in with the workaround for missing Python 3.6 made @bartoszbetka and the branch that adds Golem GUI to the machine. Please put your change directly on master and use that in the pull request.

**Comment by rwrzesien** [comment]
@cameel done :+1:

File: concent-vm/Vagrantfile [hunk 1]
```diff
@@ -12,7 +12,8 @@ Vagrant.configure("2") do |config|
 
     config.vm.network "private_network", ip: "172.40.2.3"
 
-    golem_configuration_directory = './disk/golem_configuration_directory.vdi'
+    # Path must be absolute to work correctly on Ubuntu with Vagrant 2.2.1.
+    golem_configuration_directory = File.expand_path('./disk/golem_configuration_directory.vdi')
```
Comment (cameel) [inline concent-vm/Vagrantfile]: I'd rephrase this comment because now it sounds like "it doesn't work on Ubuntu with Vagrant 2.2.1 but I won't tell you why" :)
``` python
# This should be an absolute path because `vboxmanage` fails when used with
# a relative one on some systems (discovered on Ubuntu xx.xx with Vagrant 2.2.1).
```

Comment (cameel) [reply concent-vm/Vagrantfile]: @rwrzesien What about my remark here?

Comment (rwrzesien) [reply concent-vm/Vagrantfile]: I think I must have missed it, should I add it now?

Comment (cameel) [reply concent-vm/Vagrantfile]: I have already added that change in #325. You could review it :)
